Add K3s Hardening Guide and Self Assessment (#2882)
rancher/docs: content/k3s/latest/en/security/_index.md

---
title: "Security"
weight: 90
---

> This section describes methodology and means of security a K3s cluster. It's broken into 2 sections.

Reviewer: ```suggestion
This section describes the methodology and means of securing a K3s cluster. It's broken into 2 sections.
```

> * [Hardening Guide](../hardening_guide/)

Reviewer: ```suggestion
* [Hardening Guide](./hardening_guide/)
```

> * [CIS Benchmark Self-Assessment Guide](../self_assessment/)

Reviewer: ```suggestion
* [CIS Benchmark Self-Assessment Guide](./self_assessment/)
```
